data/installed-tests/fwupd.sh

```diff
@@ -12,7 +12,7 @@ run_test()
 
 run_device_tests()
 {
-	if [ -d @devicetestdir@ ]; then
+	if [ -n "$CI_NETWORK" ] && [ -d @devicetestdir@ ]; then
```

Comment:
We're sure CI_NETWORK is set for PRs?

Reply:
It's set in the docker containers that everything runs:
```
$ grep CI_NETWORK contrib/ -R
contrib/ci/Dockerfile-ubuntu.in:ENV CI_NETWORK true
contrib/ci/Dockerfile-void.in:ENV CI_NETWORK true
contrib/ci/get_test_firmware.sh:if [ "$CI_NETWORK" = "true" ]; then
contrib/ci/Dockerfile-arch.in:ENV CI_NETWORK true
contrib/ci/Dockerfile-debian.in:ENV CI_NETWORK true
```

Reply:
So there are "some" tests (not launched in docker) that won't run with it set this means, but there will definitely be coverage.